app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -111,6 +111,11 @@
             label: logo_link_title,
           }
         } do %>
+          <% if hide_logo_text %>
+            <span class="govuk-visually-hidden">
+              <%= logo_link_title %>
```

Comment:
From looking at the issue Anika raised, I think this should use `logo_text` instead, this would change the text from "Go to the GOV.UK homepage" to "GOV.UK".